heydoctor/molekule
# chore: Convert all components to Typescript

### 📃 Summary
Move some test util files to source and convert files to typescript.

### 🔍 Changeset

### 🏷 Asana Task

### ↩️ Depends On

### 📸 Screenshots

### 🧪 QA
- Make sure build is working still
- Use storybook+chromatic to make sure no components changed
	
### 📈 Risk Analysis
- [ ] Does this affect patients?
- [ ] Does this affect clinical?
- [ ] Does this mutate data?
- [ ] Does it touch PHI?
- [ ] Does it touch payments?
- [ ] Does it update Node packages?
